Source/WebKit/UIProcess/API/gtk/DropTarget.h

```diff
@@ -83,6 +83,8 @@ class DropTarget {
     std::optional<WebCore::DragOperation> m_operation;
 #if USE(GTK4)
     GRefPtr<GCancellable> m_cancellable;
+    Vector<GRefPtr<GFile>> m_transferredFiles;
+    StringBuilder m_uriListBuilder;
```

Comment:
I really dislike this. Suggestions welcomed.

I tried adding `SelectionData::addGFiles` but for apparently there *must* be a corresponding `file://` URI for every reported file.